ontimize/components/input/combo/o-combo.component.ts

```diff
@@ -191,11 +205,17 @@ export class OComboComponent extends OFormServiceComponent implements OnInit, Af
   public clearValue(): void {
     if (this.multiple) {
       this.setValue(this.defaultValue);
+      this.value.value = [];
     } else {
       super.clearValue();
     }
   }
 
+  get showClearButton(): boolean {
+    let emptyValue = this.multiple ? this.value.value.length : Util.isDefined(this.value.value);
+    return this.clearButton && !this.isReadOnly && this.enabled && emptyValue;
```

Comment:
The name of `emptyValue` variable is confusing, we can use something like `hasValue` or `isEmptyValue`